man/genhtml.1

```diff
@@ -163,40 +363,343 @@ file view. One containing the standard information plus a link to a
 which test case covered how many lines of each source file.
 
 .RE
-.BI "\-b " baseline\-file
+.BI "\-b " baseline\-trace\-file
 .br
-.BI "\-\-baseline\-file " baseline\-file
+.BI "\-\-baseline\-file " baseline\-trace\-file
 .RS
 Use data in
-.I baseline\-file
+.I baseline\-trace\-file
 as coverage baseline.
 
 The tracefile specified by
-.I baseline\-file
-is read and all counts found in the original
-.I tracefile
-are decremented by the corresponding counts in 
-.I baseline\-file
-before creating any output.
+.I baseline\-trace\-file
+ is read and used as the baseline for classifying the change in coverage represented by the coverage counts in
+.IR tracefile .
+
+Note that you must specfiy
+.I \-\-diff\-file unfied\-diff\-file
+when you specify
+.I \-\-baseline\-file .
+Both are needed for differential coverage categorization.
+
+.RE
+.BI "\-\-baseline\-title " baseline\-title\-string
+.RS
+Use the
+.I baseline\-title\-string
+as the descriptive label for the source of coverage baseline data.
+
+.RE
+.BI "\-\-baseline\-date " baseline\-date\-string
+.RS
+Use the
+.I baseline\-date\-string
+as the collection date for the coverage baseline data.
+If this argument is not specified, the default is to use the creation time of the
+.I baseline\-trace\-file
+as the baseline date.
+
+.RE
+.BI "\-\-current\-date " current\-date\-string
+.RS
+Use the
+.I current\-date\-string
+as the collection date for the coverage baseline data.
+If this argument is not specified, the default is to use the creation time of the
+.I current\-trace\-file.
+
+.RE
+.BI "\-\-diff\-file " unified\-diff\-file
+.RS
+Use the
+.I unified\-diff\-file
+as the definition for source file changes between the sample points for
+.I baseline\-trace\-file
+and
+.IR tracefile(s) .
+A suitable
+.I unified\-diff\-file
+can be generated using the command "git diff \-\-relative SHA_base SHA_current", or using the "p4udiff" or "gitdiff" sample scripts (found in the share/lcov/support\-scripts directory shipped as part of this release).
+"p4udiff" accepts either a changelist ID or the literal string "sandbox"; "sandbox" indicates that there are modified files which have not been checked in.
+
+These scripts post\-process the 'p4' or 'git' output to (optionally) remove files that are not of interest and to explicitly not files whcih have not changed.  It is useful to note unchanged files (denoted by lines of the form
```

Comment:
```suggestion
These scripts post\-process the 'p4' or 'git' output to (optionally) remove files that are not of interest and to explicitly not files which have not changed.  It is useful to note unchanged files (denoted by lines of the form
```

Reply:
thanks...fixed in my local sandbox.
Will push this (with some other fixes) after testing is complete.